modules/ncurses/6.4.20221231/patches/add_build_file.patch

```diff
@@ -0,0 +1,430 @@
+diff --git BUILD.bazel BUILD.bazel
+new file mode 100644
+index 00000000..fcaad983
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,424 @@
++# Copyright 2020 Google LLC
```

Comment:
Could you link upstream for this file? It's probably the most complex one we would be adding to the registry. Do you think that it's sufficiently generic?

Reply:
This comes from [here](https://github.com/hdl/bazel_rules_hdl/blob/main/dependency_support/net_invisible_island_ncurses/bundled.BUILD.bazel).

Reply:
ping @fmeum . are we good to merge?